app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityTest.kt

```diff
@@ -193,6 +195,68 @@ class AdministratorControlsActivityTest {
     }
   }
 
+  @Test
+  fun testAdministratorControlsFragment_editAccountOptionsEnabled_generalOptionsIsDisplayed() {
+
```

Comment:
Nit: This blank line can be removed.

Reply:
done, thanks

Reply:
Looks to be done now. Marking as resolved.